external/glfw/lib/android/android_window.c

```diff
@@ -486,27 +486,26 @@ void _glfwPlatformPollEvents( void )
 // Called from the looper thread
 void glfwAndroidPollEvents()
```

Comment:
this function called from the looper thread in a loop: https://github.com/defold/defold/blob/2b2b6c302c9645d1ad28f95a6cd1be41324dda7c/engine/engine/src/engine_main.cpp#L131-L140

That's why I didn't add one more internal loop here